### PR TITLE
Bump cats to 1.0.0-MF

### PR DESCRIPTION
**Pull request by BenFradet**
My bad, I forgot I had specified the cats version in the github4s build

### CODE REVIEW

File: core/src/main/scala/sbtorgpolicies/libraries.scala [hunk 1]
```diff
@@ -185,7 +185,6 @@ object libraries {
     "cats-kernel"            -> (("org.typelevel", "cats-kernel", v("cats"))),
     "cats-laws"              -> (("org.typelevel", "cats-laws", v("cats"))),
     "cats-macros"            -> (("org.typelevel", "cats-macros", v("cats"))),
-    "cats"                   -> (("org.typelevel", "cats", v("cats"))),
```
Comment (juanpedromoreno) [inline core/src/main/scala/sbtorgpolicies/libraries.scala]: We could leave this key in the Map, just for legacy reasons, pointing to the `cats-core` dependency, what do you think?

```diff
- "cats"                   -> (("org.typelevel", "cats", v("cats"))),
+ "cats"                   -> (("org.typelevel", "cats-core", v("cats"))),
```

Comment (BenFradet) [reply core/src/main/scala/sbtorgpolicies/libraries.scala]: The problem is that the `cats` artifact used to contain:

- `cats-core`
- `cats-macros`
- `cats-free`
- `simulacrum`
- `machinist`

Are we okay with reducing that to only cats-core?

Comment (juanpedromoreno) [reply core/src/main/scala/sbtorgpolicies/libraries.scala]: Yeah, that's something users might not expect either. Let's remove it then, users will need to specify the new dependencies accordingly.

**Comment by BenFradet** [comment]
Thanks!

**Pull request merged**